src/client/datascience/jupyter/kernels/helpers.ts

```diff
@@ -668,7 +668,7 @@ export function findPreferredKernel(
 
                 // Find a kernel spec that matches the language in the notebook metadata.
                 if (score <= 0 && nbMetadataLanguage && speclanguage === (nbMetadataLanguage || '')) {
-                    traceInfo(
+                    traceVerbose(
```

Comment:
We haven't had issues with kernel selection (at all), hence turning them into verbose.
Reduces the amount of stuff logged